### ISSUE TITLE
script finding current implementations misreads vbnet

### ISSUE THREAD

**Issue by DLotts** [coding-horror/basic-computer-games#664]
> @MartinThoma  ...script which can automatically generate a markdown file with the current implementations...

TLDR: vbnet games are falsely marked complete.  Ignore two files: *.sln and *.vbproj

Script output is here:    

   https://github.com/coding-horror/basic-computer-games/blob/main/00_Utilities/TODO.md

Apparently, vbnet games are incomplete.  At least "Nim" is incomplete.  I was curious how much needed to change for modern basic, thinking it was probably a trivial effort.  When I looked into the Nim/vbnet folder, I see two shortish metadata files and no basic code anywhere.  Examining the merge that created the project files, I see that someone did a great job of creating them for every game in the vbnet folder.  So if you can amend your script to skip these files:

    *.sln
    *.vbproj

(merge comment: Add sln and csroj/vbproj files where missing)

Your script is quite good, thanks for doing it.
david.
